### PR TITLE
Creates jaeger config.

### LINKED ISSUE #6: Jaeger client config from config service
`TracerService` use config service to init Jaeger client.

### PR DESCRIPTION
**Pull request by dmitraver**
closes #6

### CODE REVIEW

File: src/main/scala/com/wanari/tutelar/core/ConfigService.scala [hunk 1]
```diff
@@ -36,4 +37,5 @@ trait ConfigService {
   implicit def ldapConfig: LdapConfig
   implicit def passwordSettings: PasswordSettings
   implicit def escherConfig: EscherConfig
+  implicit def jaegerConfig: Configuration
```
Comment (tg44) [inline src/main/scala/com/wanari/tutelar/core/ConfigService.scala]: I love to hate when a lib calls its inner things so common like `Configuration`.
Can we have a typealias to that somewhere like `type JaegerConfig = io.jaegertracing.Configuration` and use `JaegerConfig` in our codebase everywhere?

Comment (dmitraver) [reply src/main/scala/com/wanari/tutelar/core/ConfigService.scala]: I had exactly the same thoughts when I saw the ConfigService traitl. I was actually thinking about wrapping it in the class. Would you consider that an option?

Comment (Ksisu) [reply src/main/scala/com/wanari/tutelar/core/ConfigService.scala]: I think the simple typealias is enough (but the wrapper is fine too).
And the typealias or the wrapper should be in the `TracerService` companion object.

Comment (dmitraver) [reply src/main/scala/com/wanari/tutelar/core/ConfigService.scala]: updated my PR

**Comment by Ksisu** [comment]
Awesome! Thank you very much!

**Pull request merged**